src/main/java/Controller/Controller.java

```diff
@@ -0,0 +1,78 @@
+package Controller;
+
+import Model.Bridge;
+import Model.Player;
+import Model.Rewards;
+import view.InputView;
+import view.OutputView;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import Model.LadderDescentService;
+import Model.LadderResult;
+
+public class Controller {
+    InputView inputView = new InputView();
+    OutputView outputView = new OutputView();
+
+    public void startLadder() {
+        int height;
+
+        Player players = getPlayer();
+        Rewards rewards = getRewards();
+
+        outputView.askLadderHeight();
+        height = inputView.getWidthAndHeight();
```

Comment:
맨위에 있는 height는 여기랑 합치면 좋아보여요~

```suggestion
        int height = inputView.getWidthAndHeight();
```

Reply:
수정하겠습니다!